submissions/merowing/js-post-oop/index.js

```diff
@@ -0,0 +1,94 @@
+import { print } from './js/lib.js';
+
+class Creature {
+   constructor(name, gender, species, hands) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.hands = hands;
+   }
+
+   message() {
+      return `${this.species}; <strong>${this.name}</strong>; ${this.gender}; ${this.hands};`;
+   }
+}
+
+class Human extends Creature {
+   constructor(name, gender, hands, legs, say, species) {
+      super(name, gender, species, hands);
+      this.say = say;
+      this.legs = legs;
+   }
+
+   set friends(names) {
+      this._friends = names.map(friend => friend.name);
+   }
+
+   get friends() {
+      return ` <strong>${this._friends}</strong>`;
+   }
+
+   message() {
+      return `${super.message()} ${this.legs}; <em>${this.say}</em>;${this.friends}`;
```

Comment:
Looks like a space is missing. That's why we use less error prone techniques.
Make this fragment DRY

Reply:
done